starknet_py/tests/e2e/fixtures/contracts_v1.py

```diff
@@ -41,6 +42,27 @@ async def cairo1_erc20_class_hash(account: BaseAccount) -> int:
     return class_hash
 
 
+@pytest_asyncio.fixture(scope="package")
+async def constructor_with_arguments_class_hash(account: BaseAccount) -> int:
```

Comment:
nit: Imo it's easier to read fixtures if a function name follows the same convention as usual, so just explain what it does. For example here in can be `declare_constructor_with_arguments_contract`, but apart from that we can specify `name="..."` in `@pytest_asyncio.fixture` annotation above to simplify using a fixture in tests.

Reply:
I can agree that it is easier to read fixtures, but I chose this path on purpose. If we specify `name="..."` in the fixture filed it is not possible to find usages in convenient way. If you use fixture by function name IDE shows number of usages that was really helpful while refactoring tests